frontend/src/App.js

```diff
@@ -2,23 +2,46 @@ import React, { useState, useEffect } from 'react'
 import './App.css'
 import MyCalendar from './MyCalendar'
 import apiService from './services/apiService'
+/*
+** GRAPHQL CLIENT
+** Remove comments when ready to implement
+
+import {
+  useQuery,
+  useApolloClient,
+  useMutation,
+  useSubscription,
+} from '@apollo/client'
+import { LOGIN } from './graphql/queries'
+*/
 
 const App = () => {
 
   const [events, setEvents] = useState([])
-
+  // GraphQL Client variables
+  // const client = useApolloClient()
+  // const [token, setToken] = useState(null)
+  // END GraphQL Client variables
 
   useEffect(() => {
     apiService.getEvents().then(data => setEvents(data))
   }, [])
 
+  /* USE TOKEN IF EXISTS
+  useEffect(() => {
+    const token = localStorage.getItem('libraryapp-token')
+    if (token) {
+      setToken(token)
+    }
+  }, [])
+
+  */
+
```

Comment:
If token exists in LocalStorage, use it in the headers for the queries. Btw, change the item name from 'libraryapp-token' to something else!